web/assets/styles/themes/material.css

```diff
@@ -0,0 +1,56 @@
+:root {
+    /* Basic HABpanel styles */
+    --body-bg: #f5f5f5;
+    --body-color: #616161;
+    --primary-color: #ff3333;
+    --box-bg: #fff;
+    --header-bg: var(--primary-color);
+    /* General widget styles */
+    --widget-design-bg: #abc;
+    --widget-text-color: #333;
+    --widget-border-radius: 0px;
+    --widget-box-shadow: 0 1px 3px 0 rgba(0, 0, 0, .2), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12);
+    /* Icon colors */
+    --icon-color-filter: invert(75%) sepia(100%) hue-rotate(0deg) saturate(10000%);
+    /* Should be aligned with --primary-color */
+    --icon-color-off-filter: invert(45%) sepia(80%) hue-rotate(170deg);
+    --icon-backdrop-filter: invert(60%) sepia(100%) hue-rotate(165deg) opacity(15%);
+    /* Chart widget */
+    --chart-stroke: #456;
+    --chart-fill: #89a;
+    --chart-tooltip: #123;
```

Comment:
Could you make this white?
![image](https://user-images.githubusercontent.com/2004147/27194156-d61776c8-5201-11e7-9919-134365588d7d.png)
I know, it's probably not your fault, it was like this before too... 😊